src/Common/src/Interop/Unix/libc/Interop.mountpoints.cs

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Runtime.InteropServices;
+
+using uid_t = System.UInt32;
+
+internal static partial class Interop
+{
+    internal static partial class libc
+    {
+        [DllImport("libc")]
+        static extern int printf(string data);
```

Comment:
Same as below...I might not have pushed to origin before the PR, which would have removed these. Let me check